python3-bs4.json

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "python3-bs4",
+    "buildsystem": "simple",
+    "build-commands": [
+        "pip3 install --no-index --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} bs4"
+    ],
+    "sources": [
+        {
+            "type": "file",
+            "url": "https://files.pythonhosted.org/packages/88/df/86bffad6309f74f3ff85ea69344a078fc30003270c8df6894fca7a3c72ff/beautifulsoup4-4.6.3.tar.gz",
+            "sha256": "90f8e61121d6ae58362ce3bed8cd997efb00c914eae0ff3d363c32f9a9822d10"
+        },
+        {
+            "type": "file",
+            "url": "https://files.pythonhosted.org/packages/10/ed/7e8b97591f6f456174139ec089c769f89a94a1a4025fe967691de971f314/bs4-0.0.1.tar.gz",
```

Comment:
Just remove this.

The module you want to install is beautifulsoup4, bs4 is just a dummy package to avoid name squatting.

(that was part of an interesting experiment years ago, where someone squatted easy typos on PyPI, Rubygems, NPM, … with packages which would send info to their servers about the IP of the machine installing their packages, the uid running the install, etc… very interesting study in how all those package managers are vulnerable :slightly_smiling_face:)

Reply:
Will do. :slightly_smiling_face:

Reply:
dae/anki#257